### PR TITLE
User story 2

### PR DESCRIPTION
**Pull request by Vera-Ateeq**
User story 2 front end + backend working

### CODE REVIEW

File: client/components/Index.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import {Redirect} from 'react-router-dom'
+
+const Index = props => {
+  return <Redirect to="/pets" />
```
Comment (dovakeen118) [inline client/components/Index.js]: Not sure what this component is being used for? If it's not used we can remove it.
However, for example if we want to redirect the user from the root path to "/pets", we can add that logic to the BrowserRouter by adding an additional route

```
<Route exact path="/">
  <Redirect to="/pets" />
</Route>
```

File: client/components/PetTile.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+// leaving off here - PetTile untested and not yet built out
+// end of day one (4/19) Ben and Emilia
+
```
Comment (dovakeen118) [inline client/components/PetTile.js]: comments can be removed from final code!

File: client/components/PetTile.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  const { name, age, imgUrl, vaccinationStatus } = props.pet
+  let vaccinated = "No"
+  if (vaccinationStatus) {
+    vaccinated = "Yes"
+  }
+
+  return (
+    <div>
+      <h3>{name}</h3>
+      <img src={`${imgUrl}`} />
```
Comment (dovakeen118) [inline client/components/PetTile.js]: adding `alt` attribute to images would be a nice touch for accessibility

File: server/models/PetType.js [hunk 4]
```diff
@@ -15,14 +15,41 @@ class PetType {
...
+      const petsData = result.rows
+      const pets = petsData.map(pet => new Pet(pet))
+      return pets
     } catch (error) {
       console.log(error)
       throw (error)
     }
   }
+
+  static async findPetType(type) {
```
Comment (dovakeen118) [inline server/models/PetType.js:40]: Might consider renaming this method `findByPetType` or `findByType`

**Review by dovakeen118 — COMMENTED** [review]
Nice scope to this PR! In the future I would recommend naming this branch something relating to the feature - `user-story-2` is not very descriptive without checking all of your user stories

This looks great, please take a look at the comment for your `Index` component

File: server/models/PetType.js (hunk 4 above)
Comment (Vera-Ateeq) [reply server/models/PetType.js:40]: since this returns a PetType object, I think that findPetType is an ok name. Does it make sense?

Comment (dovakeen118) [reply server/models/PetType.js:40]: `findPetType` to me reads as "find _the_ pet type", renaming to something like `findByPetType` or `findByType` is similar to other methods we create like `findById`

**Pull request merged**